app/upgrades/v2/upgrades_test.go

```diff
@@ -0,0 +1,97 @@
+package v2_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+	"github.com/tharsis/evmos/v2/app"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	abci "github.com/tendermint/tendermint/abci/types"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+	feemarkettypes "github.com/tharsis/ethermint/x/feemarket/types"
+)
+
+type UpgradeTestSuite struct {
+	suite.Suite
+
+	ctx sdk.Context
+	app *app.Evmos
+}
+
+func (suite *UpgradeTestSuite) SetupTest() {
+	// consensus key
+	// consAddress := sdk.ConsAddress(tests.GenerateAddress().Bytes())
+
+	// setup feemarketGenesis params
+	feemarketGenesis := feemarkettypes.DefaultGenesisState()
+	feemarketGenesis.Params.EnableHeight = 1
+	feemarketGenesis.Params.NoBaseFee = false
+
+	suite.app = app.Setup(false, feemarketGenesis)
+	suite.ctx = suite.app.BaseApp.NewContext(false, tmproto.Header{
+		ChainID: "evmos_9001-2",
+	})
+}
+
+func TestUpgradeTestSuite(t *testing.T) {
+	suite.Run(t, new(UpgradeTestSuite))
+}
+
+const upgradeHeight = 58700
```

Comment:
This block height is probably too high given you are running the app

Reply:
That's correct, as we discussed on slack. You can just set the block height in the context and it doesn't matter for this test.